cmd/db_populator/main.go

```diff
@@ -40,10 +40,10 @@ func main() {
 func createSampleQuiz(db *sql.DB, title string) {
 	var quiz_id uuid.UUID
 	row := db.QueryRow(
-		`INSERT INTO quizzes (title, available_from, available_to)
-		values ($1, $2, $3)
+		`INSERT INTO quizzes (title, available_from, available_to, image_url)
```

Comment:
this was actually broken by ae2b5b5, please compile and run populate scripts before merging